### ISSUE TITLE
Hi, I have joined freecodecamp yesterday and completed 76 tasks but today all of them are disappeared and it is only showing me one. Please help. 

### ISSUE THREAD

**Issue by pc9795** [freeCodeCamp/freeCodeCamp#14355]
#### Browser Information

* Browser Name, Version: 
* Operating System: 
* Mobile, Desktop, or Tablet: 

#### Your Code

```js



```

**Comment by erictleung** [comment]
@Hitman9795 thanks for the issue. When you say "all of them disappeared", can you elaborate on that? What on the page indicates they've all disappeared? Did you go to the challenge itself and your code is gone? Or are your "brownie points" in the corner gone?

**Comment by nicosullivan** [comment]
I'm having what I believe to be the same issue. 

The count in the upper right corner of the page was reset back to one(I did a challenge and it is now two). When viewing the map only the one I just did today is showing as done. 

Below is a pic of my map

<img width="644" alt="screen shot 2017-04-12 at 10 12 02 pm" src="https://cloud.githubusercontent.com/assets/10079750/24988690/23ad5f9a-1fcd-11e7-9db5-730e82d04fda.png">

Yesterday I did all the challenges through "Create a Set of Radio Buttons"

I opened my profile in a incognito session and it only shows my task done today.


If relevant:
macOS: 10.12.3
Chrome:  57.0.2987.133 (64-bit)
